main.js

```diff
@@ -31,15 +31,50 @@ menuLinks.forEach((link) => {
   });
 });
 
-window.addEventListener('resize', () => {
-  const query = window.matchMedia('(min-width: 430px)');
-  if (query.matches) {
-    img.setAttribute('data-toggled', 'false');
-    img.src = 'images/burger-menu.png';
-    menu.classList.remove('toggled-menu');
-    menuLinks.forEach((link) => {
-      link.classList.remove('nav-link');
-    });
-    body.classList.remove('overflow');
-  }
-}, true);
\ No newline at end of file
+window.addEventListener(
+  'resize',
+  () => {
+    const query = window.matchMedia('(min-width: 430px)');
+    if (query.matches) {
+      img.setAttribute('data-toggled', 'false');
+      img.src = 'images/burger-menu.png';
+      menu.classList.remove('toggled-menu');
+      menuLinks.forEach((link) => {
+        link.classList.remove('nav-link');
+      });
+      body.classList.remove('overflow');
+    }
+  },
+  true,
+);
+
+const email = document.querySelector('#email');
+const username = document.querySelector('#fullname');
+const msg = document.querySelector('#message');
+const userInfo = {};
+
+window.addEventListener('load', () => {
+  localStorage.getItem('user_info');
+  const userInfoDeserilized = JSON.parse(localStorage.getItem('user_info'));
+  username.value = userInfoDeserilized.username;
+  email.value = userInfoDeserilized.email;
+  msg.value = userInfoDeserilized.message;
```

Comment:
- [ ] Please fix the following error that appears in the console as soon as the page is loaded which prevents the message from being preserved in the local storage.
![image](https://user-images.githubusercontent.com/62290545/189389530-75fd534c-eea0-436e-b1ff-beb4274833b7.png)